BD-1405: Update Sunset policies for email doc (#3615)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/best_practices/sunset_policies.md

# Sunset policies for email

While you may be tempted to send campaigns to as many users as you can, there are situations when it's actually advantageous to stop messages to disengaged users. 

> For emails, your sending IP has a reputation score that factors in engagement, spam reporting, blacklisting, and more. You can use tools like [Sender Score](https://www.senderscore.org/ "Sender Score") or [Outlook's Smart Network Data Service](https://postmaster.live.com/snds/ "Outlook's Smart Network Data Service") to monitor your reputation score. If your reputation score is consistently low, ISP and mailbox filters might automatically sort your emails into a spam or low priority folder for all recipients, even engaged ones. Creating a sunset policy helps ensure your emails aren't delivered to inactive recipients. 

Reviewer: ```suggestion
For emails, your sending IP has a reputation score that factors in engagement, spam reporting, blocklisting, and more. You can use tools like [Sender Score](https://www.senderscore.org/ "Sender Score") or [Outlook's Smart Network Data Service](https://postmaster.live.com/snds/ "Outlook's Smart Network Data Service") to monitor your reputation score. If your reputation score is consistently low, ISP and mailbox filters might automatically sort your emails into a spam or low priority folder for all recipients, even engaged ones. Creating a sunset policy helps ensure your emails aren't delivered to inactive recipients. 
```

> Braze's segmentation filters help prevent your messaging from appearing spammy or irrelevant by letting you easily implement sunset policies for emails, push and in-app notifications, and News Feed items. Here are some things to consider when you create a sunset policy:

Reviewer: I think "spammy" might be a bit too informal. Suggest: 

```suggestion
Braze's segmentation filters help prevent your messaging from appearing like spam by letting you easily implement sunset policies for emails, push and in-app notifications, and News Feed items. Here are some things to consider when you create a sunset policy:
```

- What counts as an "unengaged" user? 
- Is engagement defined by clicks, purchases, app usage, or a combination of these behaviors? 
- How long does the lapse in engagement need to be for you to stop sending messages?
- Will you deliver any special campaigns to users before excluding them from your segments?
- Which messaging channels will your sunset policy apply to? 

> For example, if you have users who opt-in to [Apple's Mail Privacy Protection (MPP)]({{site.baseurl}}/user_guide/message_building_by_channel/email/mpp/), consider how this may impact your email campaigns and deliverability metrics and determine how to best structure your sunset policy.

Reviewer: "opt in" as a verb here 🤔 

```suggestion
For example, if you have users who opt in to [Apple's Mail Privacy Protection (MPP)]({{site.baseurl}}/user_guide/message_building_by_channel/email/mpp/), consider how this may impact your email campaigns and deliverability metrics and determine how to best structure your sunset policy.
```

To incorporate sunset policies into your campaigns, create a [segment][19] that automatically excludes users who have marked your emails as spam or have not interacted with a your messages for a certain period of time.  

> To set up these segments, choose the "Has Marked You As Spam" and/or "Last Engaged With Message" filters located under the **Marketing Activity** section in the filter dropdown. 

Reviewer: re: discussion on filters, should be code text according to style guide

```suggestion
To set up these segments, choose the `Has Marked You As Spam` and/or `Last Engaged With Message` filters located under the **Marketing Activity** section in the filter dropdown. 
```

> When you apply the "Last Engaged With Message" filter, specify the type of messaging (push, email, or in-app notification) that the user has or has not interacted with, as well as the number of days it has been since the user last interacted. After you create a segment, choose to target this segment with any [messaging channel]({{site.baseurl}}/user_guide/message_building_by_channel/).

Reviewer: ```suggestion
When you apply the `Last Engaged With Message` filter, specify the type of messaging (push, email, or in-app notification) that the user has or has not interacted with, as well as the number of days it has been since the user last interacted. After you create a segment, choose to target this segment with any [messaging channel]({{site.baseurl}}/user_guide/message_building_by_channel/).
```

![Segment Details page with the filter "Last Engaged with Message" selected.][20]

> While Braze's platform automatically stops sending emails to users who have marked you as spam, the "Has Marked You As Spam" filter allows you to also send these users targeted push messages, in-app notifications, and News Feed cards. This filter is useful for [retargeting campaigns][21]. For instance, you can send unengaged users messages or News Feed updates that remind them of the features and deals that they are missing out on when they don't open your emails.

Reviewer: ```suggestion
While Braze's platform automatically stops sending emails to users who have marked you as spam, the `Has Marked You As Spam` filter allows you to also send these users targeted push messages, in-app notifications, and News Feed cards. This filter is useful for [retargeting campaigns][21]. For instance, you can send unengaged users messages or News Feed updates that remind them of the features and deals that they are missing out on when they don't open your emails.
```
